batchflow/models/torch/layers/conv_block.py

```diff
@@ -183,13 +182,11 @@ class BaseConvBlock(nn.Module):
         'R': 'branch',
```

Comment:
Sometimse it's residual, sometimes it's branch. It's very confusing

Reply:
Changed to `branch` `everywhere